wsgiservice_restplus/namespace.py

```diff
@@ -174,7 +174,7 @@ def inherit(self, name, *specs):
         model = Model.inherit(name, *specs)
         return self.add_model(name, model)
 
-    def expect(self, *inputs, **kwargs):
+    def expect(self, *inputs, **kwargs):    #todo: to be deprecated!
```

Comment:
let's remove this #TODO comment. Either remove the method, or keep it. #TODO shouldn't be in merged code.